specs/altair/beacon-chain.md

```diff
@@ -514,15 +495,13 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
         )
         domain = compute_domain(DOMAIN_DEPOSIT)  # Fork-agnostic domain since deposits are valid across forks
         signing_root = compute_signing_root(deposit_message, domain)
-        if not bls.Verify(pubkey, signing_root, deposit.data.signature):
-            return
-
-        # Add validator and balance entries
-        state.validators.append(get_validator_from_deposit(state, deposit))
-        state.balances.append(amount)
-        state.previous_epoch_participation.append(ParticipationFlags(0b0000_0000))  # [New in Altair]
-        state.current_epoch_participation.append(ParticipationFlags(0b0000_0000))  # [New in Altair]
-        state.leak_scores.append(0)  # [New in Altair]
+        # Initialize validator if the deposit signature is valid
+        if bls.Verify(pubkey, signing_root, deposit.data.signature):
+            state.validators.append(get_validator_from_deposit(state, deposit))
+            state.balances.append(amount)
+            state.previous_epoch_participation.append(ParticipationFlags(0b0000_0000))
```

Comment:
Why do you prefer these three statements to not have the `[New in Altair]` comment?

Reply:
It's for consistency but I don't feel too strongly. We didn't add `[New in Altair]` or `[Modified in Altair]` inside other functions called with `process_epoch` (such as `process_justification_and_finalization`, `process_rewards_and_penalties`, `process_slashings`).

Reply:
I'm good either way